recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,11 @@
 microarch_level:  # [unix and x86_64]
   - 1  # [unix and x86_64]
   - 3  # [unix and x86_64]
+
+python:
+- '3.8'
+- '3.9'
+- '3.10'
+- '3.11'
+- '3.12'
+- '3.13'
```

Comment:
Hm. As I seem to be doing this badly wrong, @chrisburr, I assumed that there would be a way to override the global pinning (https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/af28e9fa6211aa96feb9abd01c4c4cfcf8161175/recipe/conda_build_config.yaml#L784-L799)

```yaml
python:
  # part of a zip_keys: python, python_impl, numpy
  - 3.9.* *_cpython
  - 3.10.* *_cpython
  - 3.11.* *_cpython
  - 3.12.* *_cpython
python_impl:
  # part of a zip_keys: python, python_impl, numpy
  - cpython
  - cpython
  - cpython
  - cpython
python_min:
  # minimum supported python version per CFEP-25
  # bump to next minor version when we drop python versions
  - '3.9'
```

to be able to get in a final build using Python 3.8 for https://github.com/conda-forge/awkward-feedstock/pull/149, but it seems like that's getting blocked. Is this possible to do?

Reply:
If you look at "This message was generated by GitHub actions workflow ru" on the rerender message and the `run-task` job in that CI pipeline you'll see the underlying error.

My guess would be that you need to account for numpy. See https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6344/files

Reply:
> My guess would be that you need to account for numpy. See https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6344/files

Thanks @chrisburr. I thought that I would be able to not have to account for it given that it isn't a direct built dependency, but that was wrong.

The good news is that this will be short lived, as the releases we're going to will then bump up to Python 3.9 (so thank you for doing this work to help even though it will be used for less than 24 hours). This is also useful for me to have an example to refer back to in the future.